admin/installation.rst

```diff
@@ -466,7 +466,7 @@ Due to limitation of Eclipse platform used to build the Management Client, only
  
  2. Download the latest version from http://www.netxms.org/download. You will need
```

Comment:
This should be updated to current situation:
- download .jar file
- run it using java -jar nxmc-xxx.jar